### PR TITLE
Better Error Message For Missing Response Field

### PR DESCRIPTION
**Pull request by MajinFro**
from consumer tech debt card... Ruby-cb-api: make error for "missing response field" better.

> Example: https://rpm.newrelic.com/accounts/780558/applications/4991347/traced_errors/f2acfc-6ffc59d7-d8a1-11e5-9062-f8bc12425498?original_error_id=8760ae-b2f65b82-d8a1-11e5-9062-f8bc12425498

>It just reads "Results" as the error message. When you look at the stack trace you can see it's a required field that's missing.

>https://github.com/careerbuilder/ruby-cb-api/blob/master/lib/cb/responses/api_response.rb#L52

>It would be really nice if it would give info about what API was called and what we were looking for. But at least say something more than just the name of the field.

### CODE REVIEW

File: lib/cb/responses/api_response.rb [hunk 1]
```diff
@@ -49,7 +49,7 @@ def raise_on_timing_parse_error
       def required_response_field(field_name, parent_hash)
         fail ArgumentError.new("field_name can't be nil!")  if field_name.nil?
         fail ArgumentError.new("parent_hash can't be nil!") if parent_hash.nil?
-        fail ExpectedResponseFieldMissing.new(field_name) unless parent_hash.key?(field_name)
+        fail ExpectedResponseFieldMissing.new("Response field missing '#{field_name}' for #{self.class.name}") unless parent_hash.key?(field_name)
```
Comment (BradHudson) [inline lib/cb/responses/api_response.rb]: https://github.com/cbdr/ruby-style-guide#pad-string-interpolation

Use spaces with interpolation. `{ self.class.name }`

Comment (MajinFro) [inline lib/cb/responses/api_response.rb]: fixed.

**Comment by Casao** [comment]
Code Climate is bust again :(

**Comment by MajinFro** [comment]
closing because i goofed this up by not working on updated code. going to get everything fixed and then reopen.

**Pull request closed without merging**